docs/sources/troubleshooting/_index.md

```diff
@@ -49,6 +49,12 @@ If you aren't getting any patterns, you can try the following fixes:
 1. Ensure pattern extraction is enabled by setting `--pattern-ingester.enabled=true` in your Loki config. [Learn about other necessary config](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/logs/access/).
 1. It is possible that no patterns were detected, although this is rare - please [open an issue on GitHub](https://github.com/grafana/explore-logs/issues/new) or [get in touch privately](https://forms.gle/1sYWCTPvD72T1dPH9) so we can see what's going on.
 
+## There is no JSON button
```

Comment:
The JSON shows up in the panel toggle switch independent if you are using Loki 3.5, the only way you'd see no JSON is if you were running an older version of Logs Drilldown. Not sure we want this section.